cacheops/invalidation.py

```diff
@@ -18,16 +20,39 @@
 __all__ = ('invalidate_obj', 'invalidate_model', 'invalidate_all', 'no_invalidation')
 
 
+def delete_invalid_caches(conj_keys):
+    for conj_key in conj_keys:
+        offset = 0
+        while True:
+            offset, cache_keys = redis_client.sscan(conj_key, cursor=offset, count=1000)
+            if cache_keys:
+                redis_client.srem(conj_key, *cache_keys)
+                redis_client.delete(*cache_keys)
+            if offset == 0:
+                break
+    redis_client.delete(*conj_keys)
+
+
 @queue_when_in_transaction
 @handle_connection_failure
 def invalidate_dict(model, obj_dict):
     if no_invalidation.active or not settings.CACHEOPS_ENABLED:
         return
     model = non_proxy(model)
-    load_script('invalidate')(args=[
+    renamed_keys = load_script('invalidate')(args=[
         model._meta.db_table,
+        '%08x' % random.randrange(16**8),
         json.dumps(obj_dict, default=str)
     ])
+    load_cleanup_fn()(renamed_keys)
+
+
+@memoize
+def load_cleanup_fn():
+    try:
+        return import_string(settings.CACHEOPS_CLEANUP_FN)
```

Comment:
CACHEOPS_CLEANUP_FN param is missing from `.conf`

Reply:
Added it.